docs/docs/manual/starting.md

```diff
@@ -80,24 +76,22 @@ This can be useful if you want to pre-select a specific number of rows from your
 
 This can also be useful if you would like to paste in a list of URLs, which you can use later to fetch the data online and build columns with. 
 
-
 #### Database (SQL)
 
 You may want to pull the latest dataset directly from an online database. This could include an online catalogue, a Wordpress or similar content management system, or a [digital repository or collection management system](https://bits.ashleyblewer.com/blog/2017/08/09/collection-management-system-collection/).
```

Comment:
It is not clear to me how the external link is relevant to SQL import, it seems to have a much broader scope. Because this functionality only works for SQL databases at the moment, I would keep this paragraph focused on this use case. Because SQL databases are rarely completely public I would not describe them as "online databases" - I would expect people to use this with a database internal to their organization, in general.

Reply:
I think it is worth sharing some definitions - users might have metadata editing access to online collections without knowing anything of the underlying architecture - but yes, this needs more elaboration.

Reply:
I think there is a difference here - the link points to a list of data sources, but they don't seem to (generally) offer access via SQL - so they won't be relevant here as far as I can tell